cancermuts/datasources.py

```diff
@@ -1254,6 +1256,14 @@ def _get_genome_allele_freq(self, mutation, gene_id):
         self.log.info("getting genome allele frequency")
         self._get_metadata(mutation, gene_id, 'gnomad_genome_allele_frequency')
 
+    def _get_popmax_exome_allele_freq(self, mutation, gene_id):
```

Comment:
can we add a docstring to new functions that we make?

I plan eventually on having the whole code documented through docstrings. I had started doing it e.g. in `core.py` or in here - then I had to stop with lack of time, but it would be good to have docs for the functions that we are adding from scratch at least. 

You can find an example of the format in `core.py` - I tried using the [`numpy` docstring style](https://numpydoc.readthedocs.io/en/latest/format.html) which is popular in scientific Python packages.